.travis.yml

```diff
@@ -4,36 +4,55 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
-    - CONDA_NPY=110  CONDA_PY=27
```

Comment:
FYI conda-forge has dropped NumPy 1.10 already. This was necessary to not overwhelm the CIs while adding support for NumPy 1.12.